code_schemes/s01e04.json

```diff
@@ -89,6 +89,38 @@
       "StringValue": "kenya_update",
       "VisibleInCoda": true
     },
+    {
+      "CodeID": "code-49808813",
+      "CodeType": "Normal",
+      "DisplayText": "following right measures",
```

Comment:
We already have a similar theme no. 3  `follow_guidelines_call_for_right_practice`.